syntax = "proto3";

message BookRequest {
    int32 id = 1;
}

message BookResponse {
    int32 id = 1;
    string title = 2;
    string author = 3;
    int32 rating = 4;
}

service Book {
    rpc getBookInfo(BookRequest) returns (BookRequest) {}
    
}